spec/spec_helper.rb

```diff
@@ -15,6 +15,7 @@
   end
   config.include FactoryGirl::Syntax::Methods
   config.include UmtsCustomMatchers
+  Dir['./spec/support/**/*_shared_example.rb'].sort.each { |f| require f }
```

Comment:
This was a snippet I got from google (as the support directory is not natively supported). I didn't really question the sort much, but I imagine it wouldn't matter too much.